src/main/java/bio/terra/cloudfiletodatastore/deltalayer/functions/DeltaLayerBackgroundFunction.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.cloudfiletodatastore.deltalayer.functions;
+
+import bio.terra.cloudevents.GCSEvent;
+import bio.terra.cloudfiletodatastore.FileCreatedMessageHarness;
+import bio.terra.cloudfiletodatastore.FileMessage;
+import bio.terra.cloudfiletodatastore.deltalayer.DeltaLayerMessageProcessor;
+import com.google.cloud.functions.BackgroundFunction;
+import com.google.cloud.functions.Context;
+import java.util.logging.Logger;
+
+/**
+ * Implementation of a GCF background function
+ * https://cloud.google.com/functions/docs/writing#background_functions This and other background
+ * functions or other message platform listeners should pull the info needed out of the message and
+ * delegate to a {@link bio.terra.cloudfiletodatastore.MessageProcessor} subclass.
+ */
+public class DeltaLayerBackgroundFunction
```

Comment:
As in previous comments, instead of a contrived event object `GCSEvent`, use `StorageObjectData` as in done in `CloudEventsHarness`. I have no objection using `BackgroudFunction` but right now it has limitations deserializing `OffsetDateTime` type in `StorageObjectData`, the `RawBackgroudFunction` is more flexible as we can use our own configured `Gson` instance.